low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -158,6 +159,42 @@ def get_psu_info(self):
             data.append(psu_dict)
         return data
 
+    def get_platform_sensors(self):
+        sensor_list = ['temperature', 'current', 'voltage']
+        sensor_data = self.build_encl_platform_sensors_data(sensor_list)
+        return sensor_data
+
+    def build_encl_platform_sensors_data(self, platform_sensors):
+        sensors_data = {}
```

Comment:
Its recommended to add docstring. (include parameters information)

Reply:
Done.